res/css/structures/_SearchBox.scss

```diff
@@ -9,60 +9,17 @@ You may obtain a copy of the License at
 
 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+widows: THOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
I have no idea how you managed this :)

Reply:
trying to search for windows, having the focus wrong, and misspelling? No idea either :smile: